docs/application/web/guides/multimedia/metadata.md

```diff
@@ -0,0 +1,43 @@
+# Metadata
+
+You can extract metadata information from multimedia files.
+
+The main features of the Metadata API include:
+
+- Extraction of a metadata from a file
+  You can [extract metadata](#extract-metadata) from multimedia files using `get()` method.
+
+
+## Prerequisites
+
+To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html), [wearable](../../api/latest/device_api/wearable/tizen/metadata.html)), the application has to request [proper storage permissions](../security/privacy-related-permissions.md).
+
+## Extract metadata
+
+1. Create a file handle using `createFileHandle()` method:
+
+   ```javascript
+   var filePath = "videos/sample_video.mp4";
+   var fileHandle = tizen.metadata.createFileHandle(filePath);
+   ```
+
+2. When the file handle is properly created metadata can be extracted:
+
+   ```javascript
+   console.log("Video duration is: " + fileHandle.get("DURATION") + " milliseconds");
+   console.log("Video title is: " + fileHandle.get("TITLE"));
+   ```
+
+3. For better performance, FileHandle does not release underlying resources after every metadata extraction. On first extraction, data is decoded, thanks to that next metadata extractions are faster.
+When all needed metadata are extracted, handle need to be released using `release()` method:
+
+   ```javascript
+   fileHandle.release();
+   ```
+
+
+## Related Information
+- Dependencies
+  - Tizen 6.0 and Higher for Mobile
+  - Tizen 6.0 and Higher for Wearable
+  - Tizen 6.0 and Higher for TV
```

Comment:
@pkosko 
I have a few doubts: 

- Is this feature also applicable to TV as well, as per line 11 its mobile & wearable only. 
- Is this feature supported from Tizen **6.0** and onwards?

Reply:
>Is this feature also applicable to TV as well, as per line 11 its mobile & wearable only.

Good question.
@pkosko When I check https://code.sec.samsung.net/jira/browse/TWDAPI-265,
the TV is also marked to be supported.
So would you please check adding the API link to TV also on line 13?


>Is this feature supported from Tizen 6.0 and onwards?

I think that's why this PR is targeting tizen_6.0_prepare branch, not master branch.
If this update is merged into tizen_6.0_prepare, it will be kept on tizen_6.0_prepare for a while.
When Tizen 6.0 is released(Maybe on Oct.), every fixes on tizen_6.0_prepare will be merged into master branch and published.

Reply:
@nawab-samsung  and @mijin85cho
line: "To access files using the Metadata API (in [mobile](../../api/latest/device_api/mobile/tizen/metadata.html) and [wearable](../../api/latest/device_api/wearable/tizen/metadata.html) applications), the application has to request [proper storage permissions](../security/privacy-related-permissions.md)." 
says about necessary privilege request using PPM API. As TV does not have PPM feature support, I didn't mention it here, because using Metadata does not require storage privilege on TV.